nncf/common/accuracy_aware_training/runner.py

```diff
@@ -142,13 +143,11 @@ def create_training_loop(self) -> TrainingRunner:
 
 class EarlyExitTrainingRunnerCreator(TrainingRunnerCreator):
     def __init__(self, accuracy_aware_training_params, compression_controller,
-                 lr_updates_needed, verbose, validate_every_n_epochs,
-                 dump_checkpoints):
+                 lr_updates_needed, verbose, dump_checkpoints):
```

Comment:
Since you touched these, I'll ask you to set up type hints here and a docstring in this PR as well, please.

Reply:
Added small typehint